Cargo.toml

```diff
@@ -95,8 +95,9 @@ members = [
 ]
 
 exclude = [
-    # Exclude the wasm crate to avoid needless cache invalidations.
+    # Exclude the wasm crates to avoid needless cache invalidations.
     "src/sql-lexer-wasm",
+    "src/sql-pretty-wasm",
```

Comment:
Hmm, this is a bit worrying. Sorry, didn't see it go in in https://github.com/MaterializeInc/materialize/pull/21721! But it looks like this approach results in a Cargo.lock file for each one of these WASM crates, and those Cargo.lock files are not inspected by Dependabot and our Rust security vulnerability checker. 

Is there an explainer about the cache invalidation issue and why excluding these crates from the workspace solves it? Can we at least have all the WASM crates be in a single workspace of their own?